samples/counter/desktop/src/main/kotlin/com/slack/circuit/sample/counter/desktop/DesktopCounterCircuit.kt

```diff
@@ -49,17 +57,61 @@ fun Counter(state: CounterState, modifier: Modifier = Modifier) {
       )
       Spacer(modifier = Modifier.height(16.dp))
       val sink = state.eventSink
-      Button(
+      Row {
+        Button(
```

Comment:
minor nit but the plus and minus buttons are backward to me. Can you please put decrement on the left and increment on the right?